connect/api/impl/ApiClientImpl.hx

```diff
@@ -18,6 +18,8 @@ import haxe.io.BytesInput;
 class ApiClientImpl extends Base implements IApiClient {
     public function syncRequest(method: String, url: String, headers: Dictionary, body: String,
         fileArg: String, fileName: String, fileContent: Blob, certificate:String) : Response {
+            Env.getLogger().error("LOGER DE MIS COJONES");
```

Comment:
I guess these two lines don't go in production code :)

Reply:
NO... I committed and pushed in a hurry before my day off to avoid lose anything... They are already removed. Thanks for de warning.